tests/cases/fourslash/refactorConvertStringOrTemplateLiteral_ToTemplateDollar.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference path='fourslash.ts' />
+
+//// const foo = "/*x*/w/*y*/ith ${dollar}"
+
+goTo.select("x", "y");
+edit.applyRefactor({
+    refactorName: "Convert string concatenation or template literal",
+    actionName: "Convert to template literal",
+    actionDescription: "Convert to template literal",
+    newContent:
+"const foo = `with $\\\\{dollar}`",

```

Comment:
the space-time continuum inside the fourslash testing works differently
```
$\\{dollar}    -> $\{dollar}
$\\\\{dollar}  -> $\\{dollar}
```
since the resulting source code change has two backlash, I have to put four backslash in newContent